src/gdb.ts

```diff
@@ -68,8 +69,8 @@ class GDBDebugSession extends MI2DebugSession {
 		this.started = false;
 		this.crashed = false;
 		this.setValuesFormattingMode(args.valuesFormatting);
-		this.miDebugger.printCalls = !!args.printCalls;
-		this.miDebugger.debugOutput = !!args.showDevDebugOutput;
+		this.miDebugger.printCalls = args.printCalls;
```

Comment:
`args` is user supplied and may be literally any JSON type. (even though it will show type errors to the user in vscode) Keep the boolean cast to make sure we don't break any type constraints in typescript from here.